components/navbar/navbar.tsx

```diff
@@ -36,7 +36,7 @@ const Navbar = () => {
 
         <ul ref={mobileMenuRef} className={styles.navItems}>
           {Menu.map((item, index) => (
-            <li>
+            <li key={index}>
               <Link href={item.url} key={index}>
```

Comment:
If we add a li here we can remove the key in the Link tag, right?

Reply:
Yes, we can. Didn't notice the key prop in the Link tag.